Sources/protoc-gen-swift/StringUtils.swift

```diff
@@ -274,3 +274,12 @@ func stringToEscapedStringLiteral(_ s: String) -> String {
   }
   return out + "\""
 }
+
+func isValidSwiftIdentifier(_ s: String) -> Bool {
```

Comment:
The name of this function is a bit misleading; the comment makes it clearer why this works, but I would fear that future readers/users might think this function does what the name says in *all* cases. What about `isProtoIdentifierValidSwiftIdentifier`, which captures the notion that the input is assumed to already be a valid protobuf identifier?